Resources/translations/sub_items.en.xliff

```diff
@@ -86,6 +81,71 @@
         <target state="new">The selected content item(s) have been sent to trash and the selected user(s) have been deleted.</target>
         <note>key: bulk_delete.success.message.users_with_nonusers</note>
       </trans-unit>
+      <trans-unit id="540d45bd030bab3e5bec8b9c1433ff4d9998c73e" resname="bulk_hide.popup.confirm">
+        <source>Hide</source>
+        <target state="new">Hide</target>
+        <note>key: bulk_hide.popup.confirm</note>
+      </trans-unit>
+      <trans-unit id="d2ab57f08d00c6f0d714b26aa81a44c8a1867903" resname="bulk_hide.popup.message">
+        <source>Are you sure you want to hide the selected location(s)?</source>
+        <target state="new">Are you sure you want to hide the selected location(s)?</target>
+        <note>key: bulk_hide.popup.message</note>
+      </trans-unit>
+      <trans-unit id="3a8fe14b551e382f3f1f20f7aed2e5de9804f619" resname="bulk_hide.error.modal.table_title">
+        <source>Locations cannot be hidden (%itemsCount%)</source>
+        <target state="new">Locations cannot be hidden (%itemsCount%)</target>
+        <note>key: bulk_hide.error.modal.table_title</note>
+      </trans-unit>
+      <trans-unit id="5e5f346e9dca66d113585207c4e7a98e87421d34" resname="bulk_hide.error.message">
+        <source>%failedCount% of the %totalCount% selected location(s) could not be hidden because you do not have proper user permissions. {{ moreInformationLink }} Please contact your Administrator to obtain permissions.</source>
+        <target state="new">%failedCount% of the %totalCount% selected location(s) could not be hidden because you do not have proper user permissions. {{ moreInformationLink }} Please contact your Administrator to obtain permissions.</target>
```

Comment:
```suggestion
        <target state="new">%failedCount% of the %totalCount% selected Location(s) could not be hidden because you do not have proper user permissions. {{ moreInformationLink }} Contact your Administrator to obtain permissions.</target>
```